From 70181573deafb922f621e25a3b80aa9b03a6d713 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.org>
Date: Sat, 23 Dec 2017 22:10:37 +0000
Subject: [PATCH 158/277] config: Add PINCTRL_MCP23S08

As of Linux 4.12, the mcp23s08 driver moved from drivers/gpio to
drivers/pinctrl. At the same time, the Kconfig symbols changed
from CONFIG_GPIO_MCP23S08 to CONFIG_PINCTRL_MCP23S08, effectively
removing the MCP23S17 from the downstream defconfigs. Restore
support by adding CONFIG_PINCTRL_MCP23S08.

See: https://github.com/raspberrypi/linux/issues/2311

Signed-off-by: Phil Elwell <phil@raspberrypi.org>
---
 arch/arm/configs/bcm2709_defconfig | 1 +
 arch/arm/configs/bcmrpi_defconfig  | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm/configs/bcm2709_defconfig b/arch/arm/configs/bcm2709_defconfig
index e3620ef6499e..3cc3a9745c13 100644
--- a/arch/arm/configs/bcm2709_defconfig
+++ b/arch/arm/configs/bcm2709_defconfig
@@ -613,6 +613,7 @@ CONFIG_SPI_SLAVE=y
 CONFIG_PPS=m
 CONFIG_PPS_CLIENT_LDISC=m
 CONFIG_PPS_CLIENT_GPIO=m
+CONFIG_PINCTRL_MCP23S08=m
 CONFIG_GPIO_SYSFS=y
 CONFIG_GPIO_BCM_EXP=y
 CONFIG_GPIO_BCM_VIRT=y
diff --git a/arch/arm/configs/bcmrpi_defconfig b/arch/arm/configs/bcmrpi_defconfig
index c1da808d40bc..4cfd4e6a3390 100644
--- a/arch/arm/configs/bcmrpi_defconfig
+++ b/arch/arm/configs/bcmrpi_defconfig
@@ -608,6 +608,7 @@ CONFIG_SPI_SLAVE=y
 CONFIG_PPS=m
 CONFIG_PPS_CLIENT_LDISC=m
 CONFIG_PPS_CLIENT_GPIO=m
+CONFIG_PINCTRL_MCP23S08=m
 CONFIG_GPIO_SYSFS=y
 CONFIG_GPIO_PCF857X=m
 CONFIG_GPIO_ARIZONA=m
-- 
2.16.1

